src/diffusers/pipelines/loaders.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from ..models.cross_attention import LoRACrossAttnProcessor
+from ..models.modeling_utils import _get_model_file
+from collections import defaultdict
+import torch
+from ..utils import (
+    DIFFUSERS_CACHE,
+    HF_HUB_OFFLINE,
+    logging,
+)
+
+
+logger = logging.get_logger(__name__)
+
+
+LORA_WEIGHT_NAME = "pytorch_lora.bin"
+
+
+class LoraUNetLoaderMixin:
```

Comment:
New loading mixin fro LoRA

Reply:
I like it! But it does feel a bit weird that this belongs to the "core" of diffusers. Would it make sense to try to make this more general so we can use it to load other extensions (maybe textual inversion weights as well, although they don't belong to the UNet)?

Reply:
Good point Pedro! A general mixin would make sense for other extensions like `hypernetworks`, but fine for me to leave it as it is for now and adapt it later once we have other extensions.